bread/layout/error.py

```diff
@@ -0,0 +1,59 @@
+from typing import Union
+
+import htmlgenerator as hg
+
+from bread.layout.components.button import Button
+from bread.utils import Link, aslayout
+
+
+@aslayout
+def error_layout(
+    request,
+    status_code: int,
+    status_title: str,
+    description: Union[str, hg.BaseElement],
+    exception: BaseException = None,
+    exception_detail: str = None,
+    redirect_link: Link = None,
```

Comment:
To simplify the signature a bit I recommend you remove ```redirect_link``` (as it is not used) and remove the ```exception``` parameter and instead always pass ```exception_details=exception.message``` when calling ```error_layout```

Reply:
(This should also allow you to simplify the ```if``` statments below a bit, which makes it easier to read and understand.

Reply:
OK. Done.